scripts/ss.sh

```diff
@@ -7,7 +7,7 @@ trap "cleanup $? $LINENO" EXIT
 # <UDF name="sudo_username" label="The limited sudo user to be created in the cluster" />
 # <UDF name="clusterheader" label="Cluster Settings" default="Yes" header="Yes">
 # <UDF name="add_ssh_keys" label="Add Account SSH Keys to All Nodes?" oneof="yes,no"  default="yes" />
-# <UDF name="cluster_size" label="Nomad cluster size" default="3" oneof="3,5,7" />
+# <UDF name="cluster_size" label="Nomad Client Count" default="3" oneof="3,5,7" />
```

Comment:
This makes sense for deploying the three clients via the server-occ, but if someone is just deploying the client-occ, then it is using *this* stackscript and not the server-occ stackscript. Therefor we still need a `cluster_size` UDF in *this* stackscript for billing calculation when deploying this alone.

Reply:
Never mind. I see, it just looks at the UDF name, not the label. Resolving this comment.